mozilla-release/mobile/android/app/src/photon/res/values/colors.xml

```diff
@@ -271,12 +271,21 @@
     <color name="pwa_confirm_btn_normal">#E60A84FF</color>
 
     <!-- Cliqz start -->
+    <!-- General -->
+    <color name="accent_color">#00aef0</color>
```

Comment:
`accent_color` is not a very good name (does not give us any context), it comes from Lightning and we should rename it consistenlty